lib/components/Layout/Header.js

```diff
@@ -13,31 +13,44 @@ import styles from './styles/header.css';
  */
 function Branding({ logo, company }) {
   if (typeof logo === 'string') {
-      return <img className={styles.brandingLogo} src={logo} alt={company} />;
-  } else if (logo === 'false') {
+    return <img className={styles.brandingLogo} src={logo} alt={company} />;
+  }
+  if (logo === false) {
     return <span className={styles.brandingText}>{company}</span>;
-  } else {
-    const Logo = logoWhite;
-    return <div className={styles.brandingLogo}><Logo height={20} /></div>
   }
+  const Logo = logo;
+  return (
+    <div className={styles.brandingLogo}>
+      <Logo height={20} />
+    </div>
+  );
 }
 
+Branding.propTypes = {
+  company: PropTypes.string,
+  logo: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+};
+
+Branding.defaultProps = {
+  company: 'Virtru',
+  logo: logoWhite,
+};
+
 /**
  * Header Component that displays content at the top of the page.
  */
 function Header({ children, company, logo, title }) {
-  const [
-    nextChildren,
-    { Menu: MenuComponents },
-  ] = expandChildren(children, [Menu]);
+  const [, { Menu: MenuComponents }] = expandChildren(children, [Menu]);
```

Comment:
I actually find the previous format more readable for destructuring. Does ESLint discourage it?